code/go/0chain.net/smartcontract/storagesc/handler.go

```diff
@@ -21,10 +22,16 @@ func (ssc *StorageSmartContract) GetBlobberHandler(ctx context.Context,
 
 	var blobberID = params.Get("blobber_id")
 	if blobberID == "" {
-		return nil, errors.New("missing 'blobber_id' URL query parameter")
+		return nil, smartcontract.WrapErrInvalidRequest(errors.New("missing 'blobber_id' URL query parameter"))
 	}
 
-	return ssc.getBlobber(blobberID, balances)
+	bl, err := ssc.getBlobber(blobberID, balances)
+	if err != nil {
+		err := smartcontract.NewError(smartcontract.FailRetrievingBlobberErr, err)
+		return nil, smartcontract.WrapErrNoResource(err)
```

Comment:
ErrNodeNotFound, ErrValueNotPresent could be NoResource errors. Other errors could be reported as internal error?

Reply:
If checking this at every error becomes too repetitive, we can use some helper function to do it.

Reply:
I've added `NewErrNoResourceOrErrInternal` method.